python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cv.py

```diff
@@ -0,0 +1,279 @@
+        Y_train, Y_test = Y[indices1], Y[indices2]
+        result = compute_sparse_least_squares_chaos(
+            X_train, Y_train, basis, totalDegree, distribution)
+        metamodel = result.getMetaModel()
+        val = ot.MetaModelValidation(X_test, Y_test, metamodel)
+        Q2_local = val.computePredictivityFactor()[0]
+        Q2_score_list.add([Q2_local])
+    Q2_score = Q2_score_list.computeMean()[0]
+    return Q2_score
+
+# %%
+# We start by generating the input and output sample. We use a sample size equal to 1000. 
+im = ishigami_function.IshigamiModel()
+im.distributionX.setDescription(["$X_0$", "$X_1$", "$X_2$"])
+im.model.setOutputDescription(["$Y$"])
+ot.RandomGenerator.SetSeed(0)
+sample_size = 1000
+X = im.distributionX.getSample(sample_size)
+print("Input sample:")
+print(X[:5])
+Y = im.model(X)
+Y.setDescription(["$Y$"])
+print("Output sample:")
+print(Y[:5])
+
+
+# %%
+# We compute a polynomial chaos decomposition with a total degree equal to 5.
+# In order to reduce the number of coefficients, we create a sparse chaos using regression and the LARS. 
+dimension = im.distributionX.getDimension()
+basis = ot.OrthogonalProductPolynomialFactory(
+    [im.distributionX.getMarginal(i) for i in range(dimension)])
+totalDegree = 5  # Polynomial degree
+result = compute_sparse_least_squares_chaos(
+    X, Y, basis, totalDegree, im.distributionX)
+metamodel = result.getMetaModel()
+
+
+# %%
+# In order to validate our polynomial chaos, we generate an extra pair of
+# input / output samples and use the :class:`~openturns.MetaModelValidation` class. 
+test_sample_size = 200  # Size of the validation design of experiments
+inputTest = im.distributionX.getSample(test_sample_size)
+outputTest = im.model(inputTest)
+validation = ot.MetaModelValidation(inputTest, outputTest, metamodel)
+Q2 = validation.computePredictivityFactor()[0]
+graph = validation.drawValidation()
+graph.setTitle("Q2=%.2f, n=%d" % (Q2, test_sample_size))
+view = otv.View(graph)
+
+
+# %%
+# The plot shows that the score is relatively high and might be satisfactory for some purposes.
+# There are however several issues with the previous procedure:
+# 
+# - It may happen that the data in the validation sample with size 200 is more
+#   difficult to fit than the data in the training dataset.
+#   In this case, the :math:`Q^2` score may be pessimistic. 
+# - It may happen that the data in the 1000 validation sample with size 200 is
+#   less difficult to fit than the data in the validation dataset. 
+#   In this case, the :math:`Q^2` score may be optimistic.
+# - We may not be able to generate an extra dataset for validation.
+#   In this case, a part of the original dataset should be used for validation.
+# - The polynomial degree may not be appropriate for this data. 
+# - The dataset may be ordered in some way, so that the split is somewhat arbitrary.
+#   One solution to circumvent this is to randomly shuffle the data.
```

Comment:
```suggestion
#   One solution to circumvent this is to randomly shuffle the data.
# This can be done using the `:class:`~openturns.KPermutationsDistribution````

I do not know how to point to the example at:

https://openturns.github.io/openturns/latest/auto_data_analysis/manage_data_and_samples/plot_randomize_sample_lines.html

but this would be convenient.